justfile

```diff
@@ -15,7 +15,12 @@ watch:
     @echo "Watching slides"
     typst w slides.typ
 
-# Compile and Run the C code
-c:
-    @echo "Compiling and Running the C code"
-    @echo "TODO!"
\ No newline at end of file
+# Compile the C code in "code/"
+check:
+    #!/usr/bin/env bash
+    echo "compiling all *.c code to output"
+    rm -r ./output;mkdir ./output
+    for FILE in $(ls ./solutions/)
+    do
+	    cc ./solutions/$FILE -o ./output/$FILE.out
+    done
```

Comment:
This is not a `check` anymore since it compiles the `solutions` directory and not the `code` directory.